test/functional/test_site/expected/index.html

```diff
@@ -536,25 +536,27 @@ <h6 class="always-index" id="level-6-header-outside-headingsearchindex-with-alwa
               </p>
             </li>
           </ol>
-          <popover id="pop:footnote1">
-            <div slot="content">
-              <p>Here is the footnote. Footnotes will appear at the bottom of the page.</p>
-            </div>
-          </popover>
+          <span id="pop:footnote1" data-mb-component-type="popover" v-b-popover.hover.top.html="popoverInnerGenerator" class="trigger">
+          <div data-mb-html-for="content">
+            <p>Here is the footnote. Footnotes will appear at the bottom of the page.</p>
+          </div>
+        </span>
 
-          <popover id="pop:footnote2">
-            <div slot="content">
-              <p>Here's one with multiple blocks.</p>
-              <p>Subsequent paragraphs are indented to show that they belong to the previous footnote.</p>
-            </div>
-          </popover>
+          <span id="pop:footnote2" data-mb-component-type="popover" v-b-popover.hover.top.html="popoverInnerGenerator" class="trigger">
+          <div data-mb-html-for="content">
+            <p>Here's one with multiple blocks.</p>
+<p>Subsequent paragraphs are indented to show that they
+belong to the previous footnote.</p>
+          </div>
+        </span>
 
-          <popover id="pop:footnote3">
-            <div slot="content">
-              <p>Inlines notes are easier to write, since you don't have to pick an identifier and move down to type the note.
-              </p>
-            </div>
-          </popover>
+          <span id="pop:footnote3" data-mb-component-type="popover" v-b-popover.hover.top.html="popoverInnerGenerator" class="trigger">
+          <div data-mb-html-for="content">
+            <p>Inlines notes are easier to write, since
+you don't have to pick an identifier and move down to type the
+note.</p>
+          </div>
+        </span>
```

Comment:
weird that the indents are incorrect, but i assume this is what `updatetest` generated, so can't really argue with this, can I?

Reply:
Yeah..